CSharp/BatchExplorer/Models/ComputeNodeModel.cs

```diff
@@ -258,7 +306,7 @@ public override List<PropertyModel> PropertyModel
         {
             try
             {
-                if (this.ComputeNode.SchedulingState != SchedulingState.Disabled)
+                if (this.ComputeNode.SchedulingState != Microsoft.Azure.Batch.Common.SchedulingState.Disabled)
```

Comment:
You could have a using statement to shorten the string though: e.g., BatchCommon.SchedulingState.Disabled

From: paselem [mailto:notifications@github.com]
Sent: Monday, February 01, 2016 2:37 PM
To: Azure/azure-batch-samples <azure-batch-samples@noreply.github.com>
Subject: Re: [azure-batch-samples] Enable Node 'enable' and 'disable' functionality (#109)


In CSharp/BatchExplorer/Models/ComputeNodeModel.cs<https://github.com/Azure/azure-batch-samples/pull/109#discussion_r51492954>:

> @@ -258,7 +306,7 @@ public override List<PropertyModel> PropertyModel

>          {

>              try

>              {

> -                if (this.ComputeNode.SchedulingState != SchedulingState.Disabled)

> +                if (this.ComputeNode.SchedulingState != Microsoft.Azure.Batch.Common.SchedulingState.Disabled)

Yes because there is a property of the same name that the compiler attempts to resolves to first.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-batch-samples/pull/109/files#r51492954>.